Rubberduck.Parsing/VBA/RubberduckParser.cs

```diff
@@ -115,7 +113,7 @@ public void Parse()
 
             foreach (var vbComponent in components)
             {
-                while (!_state.ClearDeclarations(vbComponent)) { }
+                while (!_state.ClearStateCache(vbComponent)) { }
```

Comment:
The method needed a rename, because it's doing much more than just clearing "declarations".